sqlalchemy-stubs/engine/row.pyi

```diff
@@ -20,59 +29,62 @@ KEY_OBJECTS_NO_WARN: int
 class BaseRow:
```

Comment:
someday I guess this will become generic, right?

Reply:
I hope so, but we have to improve types elsewhere first before deciding on how to make this generic